### ISSUE TITLE
Canvas has no height

### ISSUE THREAD

**Issue by mcblum** [legalthings/angular-signature#42]
So I finally figured how what our issue is, sorry to keep bugging you about it:

<img width="426" alt="screen shot 2016-11-17 at 9 05 29 pm" src="https://cloud.githubusercontent.com/assets/1761867/20417246/aee95b4a-ad09-11e6-81bf-f6d02a1c0ccc.png">

**Comment by mcblum** [comment]
Is there anything we can do to force the canvas element to expand?

**Comment by mcblum** [comment]
Ok, so I did get this to work. I modified the directive to wrap the onResize function in a 600ms $timeout. Also, I set the width and the height of the canvas to the width and the height of the parent element. Should I submit a PR or is this not exactly how you want it to work?

**Comment by ghost** [comment]
can you show your onResize code - I am getting same issue and it really buggin me :)

**Comment by jasny** [comment]
@mcblum This looks like an old issue. Did you try the master branch?

**Comment by adrogon** [comment]
Agreed it is an old issue, but you will keep having this question until a new release is provided :)

**Comment by mcblum** [comment]
@jasny I did! I pushed the project with the most recent version (master branch) and the issue actually got worse.

@hplloyd here's what I did to the `scope.onResize` function:

```javascript
scope.onResize = function () {
                    $timeout(function () {
                        var canvas = element.find('canvas')[0];
                        canvas.width = element[0].offsetWidth;
                        canvas.height = element[0].offsetHeight;
                    }, 600);
                };
```

I'm not sure if this has other consequences, I know I removed the code which compensated for pixel density but I didn't care as much if the signature looked a little blurry just as long as people could sign.

**Comment by jasny** [comment]
Released v1.0.0. Considering this solved.

@mcblum If you still have an issue with v1.0, please create a jsfiddle that demonstrates it.
